### ISSUE TITLE
Displays link to get OpenShift console

### ISSUE THREAD

**Issue by benoitf** [redhat-developer/rh-che#1832]
Hello,

It would be nice to have inside the IDE a command (of command palette) or a widget providing some information/links about the OpenShift infra used.

I'm interested in having the console link (without using https://www.eclipse.org/che/docs/che-7/hosted-che/#finding-the-cluster-where-the-hosted-che-workspace-is-running_hosted-che or without replacing some env variables values)

Another option is to always have openshift connector plug-in being there for che.openshift.io workspaces and use the link "openshift console" (which doesn't work at the moment)

**Red Hat Che version:** 

version: current

 - [x] I can reproduce it on latest official image

**Comment by ibuziuk** [comment]
@benoitf @l0rd I would opt for enabling openshift connector plugin to every single workspace on Hosted Che

**Comment by ibuziuk** [comment]
Blocked by https://github.com/eclipse/che/issues/16890

**Comment by ibuziuk** [comment]
potentially we should include the openshift-connector plugin for pre-pulling via k8s-image-puller

**Comment by ibuziuk** [comment]
Enabling the plugin seem to be pretty expensive since it requires multiple vsix:

```
Processing plugin redhat/vscode-openshift-connector/latest
  Installing plugin extension 1/4
    Downloading plugin from https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-kubernetes-tools/vscode-kubernetes-tools-1.1.0.vsix
  Installing plugin extension 2/4
    Downloading plugin from https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-yaml/vscode-yaml-0.8.0.vsix
  Installing plugin extension 3/4
    Downloading plugin from https://github.com/che-incubator/che-theia-openshift-auth/releases/download/0.0.1/che-openshift-authentication-plugin-0.0.1.vsix
  Installing plugin extension 4/4
    Downloading plugin from https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-openshift-tools/vscode-openshift-connector-0.1.5.vsix
``` 

The plugin works on Hosted Ce just fine though:
![image](https://user-images.githubusercontent.com/1461122/88550673-38e1a980-d022-11ea-946b-9259f50b4e8d.png)

**Comment by ibuziuk** [comment]
Closing as outdated
